sysbuild/Kconfig.netcore

```
@@ -12,8 +12,14 @@ config SUPPORT_NETCORE_HCI_RPMSG
 	bool
 	default y
 
+config SUPPORT_NETCORE_802154_RPMSG
+	bool
+	default y
+
 choice NETCORE
 	prompt "Netcore image"
+	default NETCORE_HCI_RPMSG if NRF_DEFAULT_BLUETOOTH && $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
+	default NETCORE_802154_RPMSG if NRF_DEFAULT_802154 && $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
```

```suggestion
	default NETCORE_HCI_RPMSG if NRF_DEFAULT_BLUETOOTH
	default NETCORE_802154_RPMSG if NRF_DEFAULT_802154
```
I think these are redundant given `depends on SUPPORT_NETCORE`.

---

Samples applications might indicate what they need but sysbuild does not know that, sysbuild has no knowledge of the SoC internals or what the configuration of images are